### PR TITLE
fix: MM URL

### PR DESCRIPTION
**Pull request by davidterpay**

### CODE REVIEW

File: config/dydx/oracle.json [hunk 1]
```diff
@@ -431,7 +431,7 @@
           "reconnectTimeout": 2000000000,
           "maxQueries": 1,
           "atomic": true,
-          "url": "https://dydx-api.polkachu.com",
+          "url": "localhost:1317",
```
Comment (Eric-Warehime) [inline config/dydx/oracle.json:434]: Is this some arbitrary localhost port?

Comment (davidterpay) [reply config/dydx/oracle.json:434]: its the rest endpoint for cosmos chains.

Comment (Eric-Warehime) [reply config/dydx/oracle.json:434]: Should we not provide a legitimate public API to fallback to as a default?

Comment (Eric-Warehime) [reply config/dydx/oracle.json:434]: > its the rest endpoint for cosmos chains.

Gotcha. I suppose it's fine to assume you're running the node locally then.

**Review by Eric-Warehime — APPROVED** [review]

**Pull request merged**